app/src/org/commcare/tasks/DumpTask.java

```diff
@@ -237,7 +236,8 @@ protected Boolean doTaskBackground(String... params) {
                             }
                         }
                     } catch(SessionUnavailableException sue) {
-                        throw sue;

```

Comment:
Oh yea sorry, totally understood what you did, just confused as to the point of what was going on before.